README.md

```diff
@@ -52,6 +55,7 @@ Warning: This role disables root-login on the target server! Please make sure yo
 |`sftp_chroot` | true | false to disable chroot for sftp|
 |`sftp_chroot_dir` | /home/%u | change default sftp chroot location|
 |`ssh_client_roaming` | false | enable experimental client roaming|
+|`sshd_moduli` | '/etc/ssh/moduli' | path to the SSH moduli file |
```

Comment:
Please use `sshd_moduli_dir`

Reply:
Shouldn't it be `sshd_moduli_file`, as it's not a directory?